pkg/repo/chartrepo_test.go

```diff
@@ -40,37 +40,22 @@ const (
 	testURL        = "http://example-charts.com"
 )
 
-func TestLoadChartRepository(t *testing.T) {
```

Comment:
This test I removed, as its purpose appears to be to test the `chart.Load()` function